routers/web/repo/pull.go

```diff
@@ -456,6 +465,11 @@ func prepareViewPullInfo(ctx *context.Context, issue *issues_model.Issue) *pull_
 		return nil
 	}
 
+	statusCheckData := &pullCommitStatusCheckData{
+		ApproveActionLink: fmt.Sprintf("%s/actions/approve-all-checks?sha=%s&redirect=%s", repo.Link(), sha, issue.Link()),
```

Comment:
If you need to build a link, then you need to correctly escape the paramters.

Actually, you can use "form-fetch-action", no need to use the fragile `redirect` parameter